src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -320,6 +332,55 @@ public DarCollection cancelDarCollectionElectionsAsChair(DarCollection collectio
     return darCollectionDAO.findDARCollectionByCollectionId(collection.getDarCollectionId());
   }
 
+  /**
+   * DarCollections with no elections, or with previously canceled elections, are valid
+   * for initiating a new set of elections. Elections in open, closed, pending, or final
+   * states are not valid.
+   *
+   * @param user The User initiating new elections for a collection
+   * @param collection The DarCollection
+   * @return The updated DarCollection
+   */
+  public DarCollection createElectionsForDarCollection(User user, DarCollection collection) {
+    final List<String> invalidStatuses = Stream.of(
+            ElectionStatus.CLOSED, ElectionStatus.OPEN, ElectionStatus.FINAL, ElectionStatus.PENDING_APPROVAL
+    ).map(ElectionStatus::getValue).collect(Collectors.toList());
+    List<String> referenceIds = collection.getDars().values().stream().map(DataAccessRequest::getReferenceId).collect(Collectors.toList());
+    if (!referenceIds.isEmpty()) {
+      List<Election> nonCanceledElections = electionDAO.findLastElectionsByReferenceIds(referenceIds)
+        .stream()
+        .filter(e -> invalidStatuses.contains(e.getStatus()))
+        .collect(Collectors.toList());
+      if (!nonCanceledElections.isEmpty()) {
+        logger.error("Non-canceled elections exist for collection: " + collection.getDarCollectionId());
+        throw new IllegalArgumentException("Non-canceled elections exist for this collection.");
+      }
+    }
+    try {
+      collectionServiceDAO.createElectionsForDarCollection(collection);
+      collection.getDars().values().forEach(dar -> {
+        Election accessElection = electionDAO.findLastElectionByReferenceIdAndType(dar.getReferenceId(), ElectionType.DATA_ACCESS.getValue());
+        if (Objects.nonNull(accessElection)) {
+          List<Vote> votes = voteDAO.findVotesByElectionId(accessElection.getElectionId());
+          try {
+            emailNotifierService.sendNewCaseMessageToList(votes, accessElection);
+          } catch (Exception e) {
+            logger.error("Unable to send new case message to DAC members for DAR: " + dar.getReferenceId());
+          }
+        } else {
+          logger.error("Did not find a created access election for DAR: " + dar.getReferenceId());
+        }
+      });
+      // TOL ... is this the right thing to do? When a chair/admin creates an election,
+      // should we really set the update user/date here? Thinking of ditching this, but if we do,
+      // then the user display could be affected where we sort by the sort date.
+      darCollectionDAO.updateDarCollection(collection.getDarCollectionId(), user.getDacUserId(), new Date());
```

Comment:
I'm especially looking for feedback on this choice.

Reply:
I'm not sure which would be the correct action here.
Once an election is created, is there a next step where the election passes or fails?
If we update the user/date there then I would think that you wouldn't need to update here since you are not changing anything about the DAR itself?

Reply:
So I guess the question is whether or not we consider election generation a Collection update? IMO I don't think that we need to. If we need to track elections generated within a specific period of time, we can look at elections for that info. There's no changes to the Collection itself (DARs are the same), it's just updates to a different model that the collection has ties to.

Reply:
Thank you @JVThomas ... I like that line of thought, will make that change.

Reply:
PR updated, PTAL, thank you!